app/src/sharedTest/java/org/oppia/app/walkthrough/WalkthroughFragment.kt

```diff
@@ -0,0 +1,43 @@
+package org.oppia.app.walkthrough
+
+import androidx.test.core.app.ActivityScenario
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.*
```

Comment:
Avoid using * in imports.

Better use these imports:
```
import androidx.test.core.app.ActivityScenario
import androidx.test.espresso.Espresso.onView
import androidx.test.espresso.action.ViewActions.click
import androidx.test.espresso.assertion.ViewAssertions.matches
import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
import androidx.test.espresso.matcher.ViewMatchers.withId
import androidx.test.ext.junit.runners.AndroidJUnit4
import org.junit.Test
import org.junit.runner.RunWith
import org.oppia.app.R
import org.oppia.app.utility.ProgressMatcher.Companion.withProgress
```

Reply:
done